### ISSUE TITLE
2018-12-17 video upload has no sound

### ISSUE THREAD

**Issue by anacrolix** [libp2p/team-mgmt#20]
The uploaded video https://www.youtube.com/watch?v=eV0fHBm9W_s has no sound.

**Comment by mgoelzer** [comment]
Hey, I'm sorry about that - my fault.  We were having Zoom recording trouble and I jumped in with Screenflow recording what it calls "Computer Audio," but I guess that feature did not work in this case.  Unfortunately, I don't think there is anything that can be done now.  I checked the MP4 file on my desktop, and it also has no sound, so this is not a YT problem.

There are async notes.  Most of the meeting was reviewing those.  At the end, a new community member introduced himself about Go WebRTC and will follow up with us on GH and IRC.  I also reminded people about the OKR meeting tomorrow (8am PT).  That's the best summary I can give.  We agreed to cancel the 12/31 meeting (New Year's eve), so next one is Jan 14.

We're switching to a totally different Zoom account with verified Youtube live streaming for 2019, so hopefully the whole process should get more predictable and automated.
